lib/rails_admin_import/formats/json_importer.rb

```diff
@@ -0,0 +1,40 @@
+module RailsAdminImport
+  module Formats
+    class JSONImporter
+      Formats.register(:json, self)
+
+      def initialize(import_model, params)
+        if params.has_key?(:file)
+          @filename = params[:file].tempfile
+        end
+        @import_model = import_model
+      end
+
+      attr_reader :filename, :error
+
+      def valid?
+        if filename.nil?
+          @error = I18n.t('admin.import.missing_file')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.